cloud/kubernetes/helm/yugabyte/templates/_helpers.tpl

```diff
@@ -59,6 +59,19 @@ Create chart name and version as used by the chart label.
   {{- end -}}
 {{- end -}}
 
+{{/*
+  Get YugaByte master addresses for PostStart
```

Comment:
What does "for PostStart" mean? Should this just be "since we would need it post startup of the cluster"?

Reply:
PostStart is the container lifecycle hook (so the k8s term for doing the operation PostStart of the container). The reason we need another template to get the master addresses is due to the fact that while running the command, we cannot have parenthesis around the env variables, but the parenthesis are required by the YB containers while setting the flags.

Reply:
Makes sense, thanks.

Reply:
Nide_js

Reply:
Nide_js

Reply:
Nide_js